### DISCUSSION TITLE
Home Assistant OS VM // v11.2 // install

### DISCUSSION THREAD

**Discussion by dflahaut** [tteck/Proxmox#2165] [Help Needed] [answered]
hi,
I wanted to install a new fresh home assistant (3 ou 4 times) and I always have the same issue :  
 _**- Starting Home Assistant OS VM...KVM virtualisation configured, but not available. Either disable in VM configuration or enable in BIOS.
[ERROR] in line 441: exit code 255: while executing command qm start $VMID**_
what should I do ?
PS: I had no problem with the v11.1 version ...

**Comment by tteck** [comment, marked as answer]
The message you're seeing _typically_ indicates that KVM virtualization is configured on your system, but it might not be available because it's not enabled in the BIOS/UEFI settings of your physical machine.

The script runs without any problems from this end. 

```
    __  __                        ___              _      __              __     ____  _____
   / / / /___  ____ ___  ___     /   |  __________(_)____/ /_____ _____  / /_   / __ \/ ___/
  / /_/ / __ \/ __ `__ \/ _ \   / /| | / ___/ ___/ / ___/ __/ __ `/ __ \/ __/  / / / /\__ \
 / __  / /_/ / / / / / /  __/  / ___ |(__  |__  ) (__  ) /_/ /_/ / / / / /_   / /_/ /___/ /
/_/ /_/\____/_/ /_/ /_/\___/  /_/  |_/____/____/_/____/\__/\__,_/_/ /_/\__/   \____//____/

Using Advanced Settings
Using HAOS Version: 11.2
Virtual Machine ID: 102
Using Machine Type: i440fx
Using Disk Cache: Write Through
Using Hostname: haos11.2
Using CPU Model: Host
Allocated Cores: 2
Allocated RAM: 4096
Using Bridge: vmbr0
Using MAC Address: AE:1A:60:07:B7:59
Using Vlan: Default
Using Interface MTU Size: Default
Start VM when completed: yes
Creating a HAOS VM using the above advanced settings
 ✓ Using local-lvm for Storage Location.
 ✓ Virtual Machine ID is 102.
 ✓ https://github.com/home-assistant/operating-system/releases/download/11.2/haos_ova-11.2.qcow2.xz
 ✓ Downloaded haos_ova-11.2.qcow2.xz
 ✓ Extracted KVM Disk Image
 ✓ Created HAOS VM (haos11.2)
 ✓ Started Home Assistant OS VM
 ✓ Completed Successfully!
```

**Reply by dflahaut** [reply]
many thanks ! it works on one of my 4 pve. 
I will try to correct that issue ...
:-(

**Comment by VVo1verine** [comment]
Hi,

I'm just getting the same error on an old Dell laptop.
Did you managed to fix your issue ?

Thx
